news/2018-07-16-cwc-2018-recap-grand-finals.md

```diff
@@ -0,0 +1,61 @@
+
+United States' captain Zak sought to exploit this by picking [Memme - Goat Fantasia](https://osu.ppy.sh/beatmapsets/634496#fruits/1346273), forcing China not only on to a pick they had shown to be weak on, but also to immediately adapt from AR 10.5 to AR 8 Hidden. This was a risky strategy that unforunately unsettled his own team more than the Chinese, who were allowed to obtain a narrow break-point victory and set the Americans on the back foot.
+
+The teams traded blows for a time, China coming out the victors in DoubleTime and HardRock, the United States on NoMod. After an enthralling eleven thousand point victory on [Morimori Atsushi - Paradigm Shift](https://osu.ppy.sh/beatmapsets/727816#fruits/1536733) where one player from both teams failed and US player chickenbible revived on the final note, the scores were perfectly poised at 5-3 to the Chinese.
+
+Unfortunately for the United States, their viable pick options were running out, and they were forced onto the previously unseen [USAO - Boss Rush](https://osu.ppy.sh/beatmapsets/602460#fruits/1272534). Clearly beginning to falter under the pressure, the misses began to creep in and China were finally able to break the Americans' NoMod dominance, setting up a final point for themselves on [KASAI HARCORES - Cycle Hit](https://osu.ppy.sh/beatmapsets/692367#fruits/1465042), securing a 7-3 victory, and forcing a crucial bracket reset.
+
+The second match would follow a much more ominous trend for the Americans, as they watched China reverse the result of two of their winning picks from match one. Staring down a 4-0 deficit, the United States opted to return to Goat Fantasia, this time paying dividends as even the near-infallible Dusk succumbed to a fail.
+
+The United States would record a second point courtesy of Guillotine's Paradigm Shift FC, but with the scoreline sitting at 6-2, China decided, perhaps ironically, to again finish proceedings with Cycle Hit and place the final nail in the coffin for the Americans. Though the fought valiantly through 19 games and over two hours at the keyboard, the United States would have to settle for second place as China reigned supreme once again.
+
+## Summary
+
+![](/wiki/shared/news/2018-07-15-cwc-2018-recap-grand-finals/podium.png)
+
+**Congratulations to China on their victory, the United States for their first ever podium placing, and Chile for their first return to the top-three since 2013.**
+
+I know I've said it a lot these past few weeks, but this really has been the most enthralling and entertaining in osu!catch history, and perhaps osu! World Cup history as a whole. The match quality was at an insanely high level, there were surprises, shocks and twists seemingly around every corner, and every match was played out in an incredible atmosphere and positive spirit.
```

Comment:
>most enthralling and entertaining in osu!catch

probably should add tournament/world cup after entertaining there, then rephrase the ending of the sentence to fit whatever you choose

Reply:
added "tournaments"